frontend/packages/knative-plugin/src/components/overview/__tests__/KnativeResourceOverviewPage.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { LoadingBox } from '@console/internal/components/utils';
+import { ResourceOverviewDetails } from '@console/internal/components/overview/resource-overview-details';
+import OverviewDetailsKnativeResourcesTab from '../OverviewDetailsKnativeResourcesTab';
+import { KnativeOverview } from '../KnativeOverview';
```

Comment:
These are not used, and you're failing a lint telling you about it.

Reply:
fixed